ui/apps/platform/src/types/productUsage.proto.ts

```diff
@@ -0,0 +1,16 @@
+export type SecuredUnitsUsage = {
+    numNodes: number;
+    numCpuUnits: number;
+};
+
+export type TimeRange = {
+    from: string;
+    to: string;
+};
+
+export type MaxSecuredUnitsUsageResponse = {
+    maxNodes: number;
+    maxNodesAt: string;
+    maxCpuUnitsAt: string;
+    maxCpuUnits: number;
```

Comment:
If this file is not generated:
```suggestion
    maxNodes: number;
    maxNodesAt: string | null;
    maxCpuUnits: number;
    maxCpuUnitsAt: string | null;
```

Reply:
Considering `null` is automatically transformed into empty string when response is handled, I don't see why we should support additional empty value here

Reply:
To make typescript happy.

Reply:
There's no warning or anything to identify that typescript is "not happy"